_utility.css

```
@@ -0,0 +1,26 @@
+.main__social h2 {
+  color: var(--primary);
+  text-align: center;
+  padding: 0px;
+  text-align: start;
+}
+
+.main__social--logo-small {
+  width: 30px;
+  height: auto;
+  text-decoration: none;
+}
+
+.main__cv--h2 {
+  display: flex;
+  justify-content: center;
+}
+
+.ul_square {
+  list-style-type: square;
+  list-style-position: inside;
+}
+
+.details__bio--f-10 {
+  font-size: 15px;
+}
```

These don't look like utility classes. Utility classes are usually short and could be added to any HTML element. If you have specific styles for `main__cv` then this would go in the `main.css` file. I added a few suggestions on how to rename those to utility classes. It's not a perfect solution though and just my opinion.
```suggestion

.h2 {
  color: var(--primary);
  text-align: center;
  padding: 0px;
  text-align: start;
}

.logo-small {
  width: 30px;
  height: auto;
  text-decoration: none;
}

.flex-center {
  display: flex;
  justify-content: center;
}

.ul_square {
  list-style-type: square;
  list-style-position: inside;
}

.font-size15 {
  font-size: 15px;
}
```